File: Javascript/1019.js
var input = `140153`;
var lines = input.split("\n");
let t = lines[0];
let hora = parseInt(t / 3600);
let min = parseInt((t % 3600) / 60);
let seg = parseInt(t % 60);
console.log(`${hora}:${min}:${seg}`);

File: Kotlin/1019.kt
import java.util.Scanner

fun main() {
    val input = Scanner(System.`in`);
    val t = input.nextInt();
    val horas = t/3600;
    val min = (t%3600)/60;
    val seg = t%60;
    println("$horas:$min:$seg");
    
}